Installation/Action/notifications.ZaakFinishedAction.action.json

```diff
@@ -44,19 +44,44 @@
     "class": "CommonGateway\\CustomerNotificationsBundle\\ActionHandler\\NotificationsHandler",
     "configuration": {
         "extraConditions": {
-            "body.resourceUrl": {
-                "commongatewaySourceRef": "source.ref",
-                "statustype": "https://afgerondStatusTypeUrl"
+            "getObjectDataConfig": {
+                "source": "https://buren.nl/source/buren.zrc.source.json",
+                "notificationProperty": "body.resourceUrl",
+                "sourceProperties": ["statustype"],
+                "getObjectDataConfig": {
+                    "forParentProperties": ["statustype"],
+                    "source": "https://buren.nl/source/buren.ztc.source.json",
+                    "sourceProperties": ["isEindstatus"]
+                }
+            },
+            "conditions": {
+                "isEindstatus": true
```

Comment:
Didn't we agree not to have multi-layered configuration?

Reply:
You are right, we did
But i do not see a better way to do this right now, this actually will be much harder to understand and read if I make this all not multi-layered 🤔